test/support/api/flows.js

```diff
@@ -101,7 +101,11 @@ Cypress.Commands.add('routeFlows', (mutator = _.identity) => {
   cy.route({
     url: '/api/flows?*',
     response() {
-      return mutator(generateData.call(this));
+      const apiData = mutator(generateData.call(this));
+
+      if (!apiData.meta) apiData.meta = { flows: { total: apiData.data.length } };
+
+      return apiData;
```

Comment:
I wasn't sure the best way to add these `meta: {}` objects.

This allows a test to set a specific `meta.flows.total` value. Or a default one will be set based on`apiData.data.length`.

My initial solution was to add this to the `generateData()` function:

```js
function generateData(patients = _.sample(this.fxPatients, 5)) {
  const data = getResource(_.sample(this.fxFlows, 10), 'flows');

  . . .

  const meta = {
    flows: {
      total: 10
    }
  }

  return {
    data,
    included,
    meta,
  };
}
```

But then `meta.flows.total: 10` wouldn't be correct for tests where the number of flows is modified via the `mutator()`. Maybe that doesn't really matter that much. Since it wouldn't result in any errors or broken tests.

Reply:
seems ok to me